lib/stimulus/engine.rb

```diff
@@ -5,7 +5,9 @@ class Engine < ::Rails::Engine
     config.autoload_once_paths = %w( #{root}/app/helpers )
 
     initializer "stimulus.assets" do
-      Rails.application.config.assets.precompile += %w( importmap.json stimulus/manifest )
+      Rails.application.config.respond_to?(:assets) do
+        Rails.application.config.assets.precompile += %w( importmap.json stimulus/manifest )
+      end
```

Comment:
This is the actual fix, right? Can only be this included in the PR and the rest reverted?

Reply:
Maybe, but as mime_magic was removed as a dependency in Rails an update to 6.1.3.1 is required and it shouldn't affect 2.7.2 but Ruby 3.0.0 requires the addition of rexml to the Gemfile.